### PR TITLE
[ADD] portal_partner_data_no_edit: New module

### PR DESCRIPTION
**Pull request by chienandalu**
This module allows to block portal users info editing to prevent unproper
changes. We can configure it on every partner.

image

cc @Tecnativa TT27796

### CODE REVIEW

File: portal_partner_data_no_edit/README.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To test this module, you need to:
+
+#. Block portal info editing for a partner:
```
Comment (pedrobaeza) [inline portal_partner_data_no_edit/README.rst:41]: This part should go on CONFIGURE.rst

**Review by pedrobaeza — COMMENTED** [review]
2 functional comments (to add already or put them in roadmap):

- Add the flag to the invite to portal popup
- Add a text on the page highlighted (for example with info BS class), saying for those blocked users that they should contact the company for any modification

File: portal_partner_data_no_edit/README.rst (hunk 1 above)
Comment (chienandalu) [reply portal_partner_data_no_edit/README.rst:41]: fixed

**Comment by chienandalu** [comment]
> 2 functional comments (to add already or put them in roadmap)

We're short on time for this task already, so I've put it on the Roadmap

**Review by pedrobaeza — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
LGTM Functional review

**Comment by pedrobaeza** [comment]
Let's go with it:

/ocabot merge nobump

**Pull request merged**